docs/src/contributing/development-environment.md

```diff
@@ -1,280 +1,266 @@
 # Development Environment
 
-[DigitalOcean][] is the recommended development environment for Deis project contributors. [Provision a new DO cluster](../installing-deis/digitalocean.md) and then continue to follow the instructions below to get started hacking.
+This document is for developers who are interested in working directly on the Deis codebase. In this guide, we walk you through the process of setting up a development environment.
 
-!!! important
-    Are you a new contributor to Deis? Your first [Pull Request][pr] could earn you credit at [DigitalOcean][]! Submit your changes and then email <deis@engineyard.com>. When your PR is merged, the maintainer team will send you a DigitalOcean credit based on the value of your contribution.
-
-This document is for developers who are interested in working directly on the Deis codebase. In this guide, we walk you through the process of setting up a local development environment. While there are many ways to set up your Deis environment, this document covers a specific setup:
+We try to make it simple to hack on Deis components. However, there are necessarily several moving pieces and some setup required. We welcome any suggestions for automating or simplifying this process.
 
-- Developing on **Mac OSX** or **Linux**
-- Managing virtualization with **Vagrant/Virtualbox**
-- Hosting a docker registry with **docker-machine** (Mac)
-
-We try to make it simple to hack on Deis. However, there are necessarily several moving pieces and some setup required. We welcome any suggestions for automating or simplifying this process.
+!!! note
+    The Deis team is actively engaged in containerizing Go and Python based development environments tailored specifcally for Deis development in order to minimize the setup required.  This work is ongoing.
```

Comment:
an (optional, of course) idea: link to some or all of the 'containerize the dev environment' issues we have on the repos so people can follow the work as it progresses? some of the repos (router, for example, as you know ;) already are fully, or almost fully, containerized